### DISCUSSION TITLE
The mobile app crashes on my note 10+ when i try to login

### DISCUSSION THREAD

**Discussion by pryme0** [community/community#63825] [Mobile] [answered]
### Select Topic Area

Bug

### Body

When I click on the signin button the app crashes on my Samsung note 10+

**Comment by eliperkins** [comment]
Hey @pryme0, thanks for this bug report.

Crashing on login sounds really frustrating. I've created an issue in our internal repo to track this, and I'll work with our engineers to get this addressed as soon as possible.

Thanks for helping us improve GitHub Mobile. 🙇

**Comment by alcere** [comment]
@pryme0 thanks for reporting! Could you share more info with us? Like the app version you are running,  if you are part of the beta group, or if it ever worked for you. All this info will be incredibly useful to find the root cause for the issue!

**Comment by alcere** [comment]
@pryme0 we added some changes recently to our sign-in flow, have you noticed any improvement?

**Reply by pryme0** [reply, marked as answer]
It works perfectly now!

Thank you!!
